app/model/application.js

```diff
@@ -185,6 +197,26 @@ ApplicationSchema.statics.syncIssuesToGitHub = function(application) {
     .then(() => application);
 }
 
+ApplicationSchema.statics.syncLabelToGitHub = function(application) {
```

Comment:
It isn't called anywhere. It should be called before an issue is synced, so I guess i'll add the code for that on this pr.

It will always return application. It runs basically the parseAppHubFile and fetchDesktopFile functions, returning an edited, unsaved, application object. I'll add some more logic for in case the POST fails.